ckks/bootstrap.go

```diff
@@ -131,31 +132,27 @@ func (btp *Bootstrapper) modUp(ct *Ciphertext) *Ciphertext {
 	return ct
 }
 
-func (btp *Bootstrapper) coeffsToSlots(vec *Ciphertext) (ct0, ct1 *Ciphertext) {
+// CoeffsToSlots applies the homomorphic encoding
+func CoeffsToSlots(vec *Ciphertext, pDFTInv []*PtDiagMatrix, eval Evaluator) (ct0, ct1 *Ciphertext) {
```

Comment:
I also would like to provide CtS and StC as part of the evaluator API (were it could use already available memory pool), but it is not as easy as it sounds because this would imply that keys for those operations should be easily created without having to use the boostrapping parameters. I propose to work on this features in a following PR.

Reply:
Fine for me to not move the CtS and StC methods in the evaluator yet.